encrypted_search/index.py

```diff
@@ -0,0 +1,69 @@
+from typing import List, Set, Tuple
+
+from nltk import word_tokenize
+from nltk.corpus import stopwords
+
+from utils.types import corpus_type, event_type, index_type
+
+
+class EncryptedIndex:
+    """A searchable obfuscated index.

```

Comment:
I understand what you're getting at when you say "obfuscated" here.  But for people in Cryptography, it has a very specific meaning that's something different from this.  And colloquially, the word tends to mean "something not really secure but sort of weakly obscured".